src/demo/styles.css

```diff
@@ -2,6 +2,38 @@
 @import 'node_modules/pikaday/css/pikaday.css';
 @import 'node_modules/pikaday/css/triangle.css';
 
```

Comment:
These styles are pulled from publish. However, they aren't able to be used in publish from the styleguide at this time. We don't have any plans currently for exporting global styles from the styleguide, which is mostly what the original webpack build was supposed to be doing for us.